server/app/services/settings/AbstractSettingsManifest.java

```diff
@@ -4,21 +4,67 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSortedMap;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigException;
 import java.util.Locale;
 import java.util.Optional;
 import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.Http;
 
 /** Provides behavior for {@link SettingsManifest}. */
 public abstract class AbstractSettingsManifest {
+  private static final String FEATURE_FLAG_SETTING_SECTION_NAME = "Feature Flags";
```

Comment:
suggestion (non-blocking): I like making this a const. I suggest using it other places we're currently using the string "Feature Flags" if it makes sense to. Specifically the `GENERATED_SECTIONS` var in `SettingsManifest.java` to provide extra assurance that that section will be found.

Reply:
That's a bit tricky given SettingsManifest.java is generated based on the contents of env-var-docs.json, where `"Feature Flags"` is one of the section titles

Reply:
Ah gotcha, never mind then.